IPython/utils/io.py

```diff
@@ -37,7 +37,7 @@ def __init__(self, stream, fallback=None):
         def clone(meth):
             return not hasattr(self, meth) and not meth.startswith('_')
         for meth in filter(clone, dir(stream)):
-            setattr(self, meth, getattr(stream, meth))
+            setattr(self, meth, getattr(stream, meth, None))
```

Comment:
I think you're right. I was going for "least invasive change" instead of what is truly right to do. I always do that when approaching large projects and I need to stop. I'm gonna fix it up.